docs/src/best-practices-js.md

```diff
@@ -80,15 +110,24 @@ Playwright has a [test generator](./codegen.md) that can generate tests and pick
 
 #### Use `codegen` to generate locators
 
-To pick a locator you can run the `codegen` command and click on the pick locator button. Then hover over your page in the browser window and click on the element you want to pick. You can then copy and paste this locator into your code. You can also use `codegen` to record a test for you.
+To pick a locator run the `codegen` command followed by the URL you would like to pick a locator from.
 
 ```bash
-npx playwright codegen
+npx playwright codegen playwright.dev
 ```
+
+This will open a new browser window as well as the Playwright inspector. To pick a locator first click on the 'Record' button to stop the recording. By default running the `codegen` command will start recording a new test based on your user actions in the browser window. 
```

Comment:
```suggestion
This will open a new browser window as well as the Playwright inspector. To pick a locator first click on the 'Record' button to stop the recording. By default running the `codegen` command will begin to record a new test based on the users actions in the browser window. 
```